test/_init_tests.js

```diff
@@ -265,13 +266,40 @@ sinon.sandbox.create = function(config) {
   return sandbox;
 };
 
+function mockConsoleError() {
+  consoleMock = consoleSandbox.mock(console);
+  consoleMock.expects('error').never();
+}
+
+function forbidConsoleError() {
+  consoleSandbox = sinon.sandbox.create();
+  mockConsoleError();
+  this.allowConsoleError = function(func) {
+    restoreConsoleError();
+    func();
+    mockConsoleError();
+  };
+}
+
+function restoreConsoleError() {
+  try {
+    consoleMock.verify();
+  } catch (e) {
+    const helpMessage =
+        'Your test resulted in an unexpected call to console.error.\n' +
```

Comment:
nit: :%s/unexpected call/uncaught call/g (since you're continuing later to say "if this was expected/not expected, it seems odd to announce from beforehand that the call was unexpected)